shaku_axum/src/inject_provided.rs

```diff
@@ -72,22 +71,22 @@ where
     M: ModuleInterface + HasProvider<I> + ?Sized,
     I: ?Sized,
 {
-    type Rejection = (StatusCode, axum::Json<Value>);
+    type Rejection = (StatusCode, String);
 
     async fn from_request(req: &mut RequestParts<B>) -> Result<Self, Self::Rejection> {
         let module = get_module_from_state::<M, B>(req)?;
         let service = module.provide().map_err(|e| {
             (
                 axum::http::StatusCode::INTERNAL_SERVER_ERROR,
-                axum::Json(json!({ "error": format!("{}", e) })),
+                format!("{}", e),
```

Comment:
nit: this can use `to_string`:
```suggestion
                e.to_string(),
```